Tasks/Burak/Task3/Task3.py

```diff
@@ -0,0 +1,65 @@
+             5:  "пять часов ровно", 6:  "шесть часов ровно", 7: "семь часов ровно", 8:  "восемь часов ровно",
+             9: "девять часов ровно", 10: "десять часов ровно", 11: "одинадцать часов ровно",
+             12: "полдень", 13: 'один час ровно', 14: "два часа ровно", 15: "три часа ровно",
+             16: "четыре часа ровно", 17:  "пять часов ровно", 18:  "шесть часов ровно", 19:  "семь часов ровно",
+             20: "восемь часов ровно", 21:  "девять часов ровно", 22: "десять часов ровно",
+             23: "одинадцать часов ровно"}
+
+dictofminutes = {1: "одна минута", 2: "две минуты", 3: "две минуты", 4:  "две минуты", 5:  "пять минут",
+                    6:  "шесть минут", 7: "семь минут", 8:  "восемь минут", 9: "девять минут", 10: "десять минут",
+                    11: "одиннадцать минут", 12: "двенадцать минут", 13: 'тринадцать минут', 14: "четыренадцать минут",
+                    15: "пятнадцать минут", 16: "шестнадцать минут", 17:  "семнадцать минут", 18:  "восемнадцать минут",
+                    19:  "девятнадцать минут", 20: "двадцать минут", 21:  "двадцать одна минута",
+                    22:  "двадцать две минуты", 23:  "двадцать три минуты", 24:  "двадцать четыре минуты",
+                    25:  "двадцать пять минут", 26:  "двадцать шесть минут", 27:  "двадцать семь минут",
+                    28:  "двадцать восемь минут", 29:  "двадцать девять минут", 31: 'тридцать одна минута ',
+                    32: 'тридцать две минуты', 33: 'тридцать три минуты', 34: 'тридцать четыре минуты',
+                    35: 'тридцать пять минут', 36: 'тридцать шесть минут', 37: 'тридцать семь минут',
+                    38: 'тридцать восемь минут', 39: 'тридцать девять минут', 40: 'сорок минут',
+                    41: 'сорок одна минута', 42: 'сорок две минуты', 43: 'сорок три минуты', 44: 'сорок четыре минуты',
+                    45: 'без пятнадцати минут', 46: 'без четырнадцати минут', 47: 'без тринадцати минут',
+                    48: 'без двенадцати минут', 49: 'без одинадцати минут', 50: 'без десяти минут',
+                    51: 'без девяти минут', 52: 'без восьми минут', 53: 'без семи минут', 54: 'без шести минут',
+                    55: 'без пяти минут', 56: 'без четырёх минут', 57: 'без трёх минут', 58: 'без двух минут',
+                    59: 'без одной минуты '}
+
+hours = ["первого", "второго", "третьего", "четвертого", "пятого", "шестого", "седьмого", "восьмого", "девятого",
+         "десятого", "одинадцатого", "двенадцатого", "первого", "второго", "третьего", "четвертого", "пятого",
+         "шестого", "седьмого", "восьмого", "девятого", "десятого", "одинадцатого", "двенадцатого"]
+dictifmorethan45= ["час", "два", "три", "четыре", "пять", "шесть", "семь", "восемь", "девять", "десять",
+                   "одинадцать", "двенадцать", "час", "два", "три", "четыре", "пять", "шесть", "семь", "восемь",
+                   "девять", "десять", "одинадцать", "двенадцать" ]
```

Comment:
the data structure is very good for the task. but it contains some duplicates and can be optimized